psc/Main.hs

```diff
@@ -143,6 +143,11 @@ sourceMaps = switch $
      long "source-maps"
   <> help "Generate source maps"
 
+dumpCoreFn :: Parser Bool
+dumpCoreFn = switch $
+     long "dump-corefn"
+  <> help "Dump CoreFn"
```

Comment:
Please replace "Dump  CoreFn" here with something like "Dump the (functional) core representation of the compiled code at output/*/corefn.json"